### PR TITLE
[CI] TimeGPT tests

### PR DESCRIPTION
**Pull request by paxcema**
## Description

Should merge after #7198

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by paxcema** [comment]
@ZoranPandovski we need to add the key as a secret for these new tests to pass.

File: tests/unit/ml_handlers/test_timegpt.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        self.run_sql(
+            """
+            SELECT p.ma
+            FROM proj.test_timegpt_forecast as p
+            JOIN pg.df as t
+            WHERE p.saledate > LATEST;
+        """
+        )
+
+        # asserts
```
Comment (ZoranPandovski) [inline tests/unit/ml_handlers/test_timegpt.py:102]: Looks great. Are we missing asserts here ?

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: tests/unit/ml_handlers/test_timegpt.py (hunk 1 above)
Comment (paxcema) [reply tests/unit/ml_handlers/test_timegpt.py:102]: Absolutely, that's a great catch! I will add some as soon as the API key gets recognized by the CI

Comment (paxcema) [reply tests/unit/ml_handlers/test_timegpt.py]: Added :white_check_mark:

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**